samples/filter_sort_groups.py

```diff
@@ -0,0 +1,86 @@
+####
+# This script demonstrates how to filter groups using the Tableau
+# Server Client.
+#
+# To run the script, you must have installed Python 2.7.9 or later.
+####
+
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Filter on groups')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+    parser.add_argument('-p', default=None)
+    args = parser.parse_args()
+
+    password = ''
```

Comment:
correct, password gets set in all cases because args.p is either `None`, or it isn't.